tasks/update_nodes.go

```diff
@@ -48,9 +48,12 @@ func (t *UpdateNodesTask) updateUsingRescueProxy() error {
 		t.logger.Warn("Failed to update node registry", zap.String("source", src), zap.Error(err))
 		return err
 	}
+	newList := make([]models.NodeID, 0, len(nodes))
 	for _, n := range nodes {
-		t.nodes.Add(common.BytesToAddress(n))
+		newList = append(newList, common.BytesToAddress(n))
 	}
+	t.nodes.Add(newList)
```

Comment:
Smart

Reply:
i almost made `Add()` variadic instead but i figure we only ever do this in bulk so it makes sense to have a bulk api